src/Project.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace LanguageServer;
+
+use PhpParser\{Error, Comment, Node, ParserFactory, NodeTraverser, Lexer};
+use PhpParser\PrettyPrinter\Standard as PrettyPrinter;
+use PhpParser\NodeVisitor\NameResolver;
+
+class Project
+{
+    private $documents;

```

Comment:
Need some `@var` annotations here and document this is an associative array of URI => document